### PR TITLE
[skyapi] refs #131 adding a deploy section to skyapi

### PR DESCRIPTION
**Pull request by stdevRulo**
adding a deploy section for skyapi using jobs to be able to work in different directories and keep deploying pyskycoin

### CODE REVIEW

**Comment by stdevMac** [comment]
@stdevRulo pls sign cla certificate

File: .travis.yml [hunk 1]
```diff
@@ -94,40 +94,59 @@ before_deploy:
...
+        password:
+          secure: Lek+1bI8jaYADO5b4oLTirI3xqdYktV/ihyGCraK+Glp/U6FwwlF3ttRpvdT9/YyVRgWEt/jePQxH+COW3WbKh54MZWCMlMKmWc4JTs6zHliAWXecAQSvPDWHjzO0F1iz839MRBfQdBUMh1g7BFi4nC886DS7ts7+p2XXGsWwtI+quOwvZ6bS0M6nMTecG8OtpYCkOAoGZI6JEoDXyIygPR5+TIWXO2QFt7KT+fsircOvDHzdA2YY/oexwUWwLLf7J0qnRm2AEx0dDjpFTCaOr6+VxdjD7F8qDoXePzVEvW7AG6k0Emx3Vuj9JiTgga8qX3ExHSXoomNhiJoOM30UbH00mdT5Fz+6HqYqFieijX/Li2fExqXXp6Za4VHMrwImrXoYQw4350F8b6QzF7E4WouBBysBZIkBEswp69783uU1lGcbU0wnyj8SN6rYIPO03MG1e5HO+Dd9avMtCMdGj3qsNlmfK0v3fAGjrYsStMQtrfOUZGTahrHPh7P3VPB4qmsJ7eNo+lFtLN0I2+ehoqMd3UH+qggdFdLQFGVMe0SWCUV9Aoyxo9vVqbnPzclsmWL44M6YRDIgOtochNiDINbdx/EA/L517GrpSDjZwvhrJkoCE7Wr9m5YcC+x8qQXCpQ4dcYAJ4C+okASAIwtjOai5dQB7TfHJfN5XzSLUI=
+        on:
+          repo: skycoin/pyskycoin
+          branches:
```
Comment (olemis) [inline .travis.yml]: see conditions in `pyskycoin` stage above .

Comment (stdevRulo) [reply .travis.yml]: the conditions are specified with the regular expression that is just below the only part

File: .travis.yml [hunk 2]
```diff
@@ -81,53 +81,13 @@ install:
 before_script:
   # Install PyPA Docker images for building multilinux wheels
 script:
+#Deploy to Pypi
+- cd ${TRAVIS_BUILD_DIR}
+- ./.travis/deploy-with-twine.sh
```
Comment (stdevMac) [inline .travis.yml]: @stdevRulo pls change script name to `deploy-skyapi.sh` or something more meaningfull

File: lib/skyapi/Makefile [hunk 3]
```diff
@@ -0,0 +1,116 @@
+.DEFAULT_GOAL := help
```
Comment (stdevMac) [inline lib/skyapi/Makefile]: Why we have a Makefile here in `lib/skyapi` move what you need to makefile in project root and make cd according to yours needs

Comment (stdevRulo) [reply lib/skyapi/Makefile]: That's a residue that's left from when using the travis deploy section. In the next commit I delete it

**Pull request closed without merging**